### PR TITLE
Add bookings controller

### PR DESCRIPTION
**Pull request by CesarCachay**
- Add crud controllers for Bookings

### CODE REVIEW

File: app/controllers/admin/bookings_controller.rb [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+    if @booking.save 
+      redirect_to bookings_path(booking), notice: 'Booking already created!'
+    else
+      render :new
+    end
   end
 
-  def edit
+  def update
+    @booking = Booking.find(params[:id])
```
Comment (cristianbgp) [inline app/controllers/admin/bookings_controller.rb]: You don't need to use this line, if you already set `@booking` in the method `set_bookings`

File: app/controllers/admin/bookings_controller.rb [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
   def show
   end
+  
+  def edit
+  end
 
-  def new
+  def create
+    @booking = Booking.create(set_bookings)
+    @booking = current_user.bookings.create(set_bookings)
```
Comment (cristianbgp) [inline app/controllers/admin/bookings_controller.rb]: Why do you need this line?

File: app/controllers/admin/bookings_controller.rb [hunk 3]
```diff
@@ -1,13 +1,53 @@
...
+
+    if @booking.save 
+      redirect_to bookings_path(booking), notice: 'Booking already created!'
```
Comment (cristianbgp) [inline app/controllers/admin/bookings_controller.rb]: I think the path would be `admin_booking_path(@booking)`

File: app/controllers/admin/bookings_controller.rb [hunk 4]
```diff
@@ -1,13 +1,53 @@
...
+    else
+      render :new
+    end
   end
 
-  def edit
+  def update
+    @booking = Booking.find(params[:id])
+    if @booking.update(booking_params)
+      redirect_to bookings_path(@booking), notice: 'Booking already updated!'
```
Comment (cristianbgp) [inline app/controllers/admin/bookings_controller.rb]: I think the path would be `admin_booking_path(@booking)`

File: app/controllers/admin/bookings_controller.rb [hunk 5]
```diff
@@ -1,13 +1,53 @@
...
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @booking.destroy
+    redirect_to bookings_path, notice: "Booking already deleted!"
```
Comment (cristianbgp) [inline app/controllers/admin/bookings_controller.rb]: I think the path would be `admin_bookings_path`

File: app/controllers/admin/bookings_controller.rb (hunk 3 above)
Comment (CesarCachay) [reply app/controllers/admin/bookings_controller.rb]: Yes, bro. I forgot we are in a namespace

**Pull request merged**